.circleci/config.yml

```diff
@@ -1,19 +1,91 @@
+e2eTest: &e2eTest
+    machine: true
+    working_directory: /home/circleci/.go_workspace/src/github.com/giantswarm/cert-exporter
```

Comment:
Maybe you could use here `$CIRCLE_PROJECT_REPONAME` to make it easier to c&p between projects?